recipe/meta.yaml

```
@@ -1,54 +1,60 @@
 {% set name = "pdoc3" %}
-{% set version = "0.9.2" %}
+{% set version = "0.11.6" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9df5d931f25f353c69c46819a3bd03ef96dd286f2a70bb1b93a23a781f91faa1
+  sha256: 1ea5e84b87a754d191fb64bf5e517ca6c50d0d84a614c1efecf6b46d290ae387
 
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<39]
   entry_points:
     - pdoc = pdoc.cli:main
     - pdoc3 = pdoc.cli:main
-  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - mako
     - markdown >=3.0
     - pip
-    - python >=3.5
+    - python
     - setuptools_scm
     - setuptools-git
   run:
     - mako
     - markdown >=3.0
-    - python >=3.5
+    - python
 
 test:
   imports:
     - pdoc
     - pdoc.cli
     - pdoc.html_helpers
-
+  source_files:
+    - pdoc/test
   commands:
-    - pdoc --help
-    - pdoc3 --help
-    # commenting at rev 0.7.2 because of an irrelevant git test failing.
-    #     https://github.com/conda-forge/staged-recipes/pull/10220#issuecomment-557137812
-    # - python -m pdoc.test.__init__
+    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
+    - PYTHONPATH=$SRC_DIR python -m unittest discover -s pdoc.test  # [not win]
+    - set PYTHONPATH=%SRC_DIR% && python -m unittest discover -s pdoc.test -v  # [win]
 
 about:
-  home: https://pdoc3.github.io/pdoc/
-  license: GNU Affero General Public v3 or later (AGPLv3+)
+  home: https://pdoc3.github.io/pdoc
+  license: GNU Affero General Public License v3 or later (AGPLv3+) (AGPL-3.0)
```

The `license` statement wants to be an [SPDX](https://spdx.org/licenses/) friendly construction.

`LICENSE.txt` says AGPLv3 and I can't immediately see that anything in the project says "or later".

That would suggest we want

```suggestion
  license: AGPL-3.0-only
```